### ISSUE TITLE
Cards with the same names are only getting processed once

### ISSUE THREAD

**Issue by berndt-toast83** [ndepaola/mpc-scryfall#6]
@ndepaola , this issue affects tokens mostly and cards from older sets where alternate art was available for a single card. What happens is, when you are processing a set, it only take one of those cards to use for that Name passed in. Thus, only card is processed and only one image is available. An easy way to see this is in the Battle for Zendikar tokens set.
https://scryfall.com/sets/tbfz?order=name
Here, we have 3 tokens for Eldrazi Scion, with different art. In practice, this is ok as they are the same token, just with new art. However, if you look at the elemental token, we have 2 distinct tokens with the same name that are completely different. One is Red 3/1 Haste, and then other is Green/Red 5/5. We only download one of these and the other is left behind. It would be great if we had a way of downloading all cards in the set, regardless of name. This would require us not overwriting the previous card if we only name the files by Name. This is not high priority by any means, but a nice to have.
Thanks,
Bryan

**Comment by berndt-toast83** [comment]
To be clear, I am less interested in the alternate art, I am more interested in getting all distinct cards if they have the same name.

**Comment by berndt-toast83** [comment]
Maybe searching on collector number, since that will be unique even if the names are the same.

**Comment by VanCityGuy** [comment]
I'd love to see support for this to. But unlike berndt-toast, I do want it for the different art, especially for the lands. I know there is unique card identifier in the scryfall search, so hopefully this could be implemented. Thanks for the awesome tool!

**Comment by VanCityGuy** [comment]
Also just to clarify, would love to see this implemented in the scryfall_formatter.py script if possible.

Thanks!

**Comment by berndt-toast83** [comment]
@ndepaola what do you think?

**Comment by berndt-toast83** [comment]
Adding this to the query string should get us the full set of all images:
card = scrython.cards.Search(q=query, unique="prints").data()[0]

This is where I need coding help to make sure that we are grabbing all of these not based on card name, but rather on some other unique identifier and then naming these duplicates in some way as to not overwrite the previous file. Here maybe?
for card_obj in cards:

**Comment by VanCityGuy** [comment]
Yeah I was able to adjust the quesry string to that (after much trial and error, my coding knowledge is very limited). It  processed without error, but the issue was as as you mentioned actually getting it to process each different version, and renaming them to providing overwriting. Hopefully @ndepaola can help when we has some free time :)

I'm hoping to process all of the lands and offering them up to your drive @berndt-toast83 as my contribution to the community.

Now we just need that Deck consolidation tool I posted about in the subreddit! :D
